### PR TITLE
[T4A2][W09-A1] Low Yong Siang

### PR DESCRIPTION
**Pull request by YongSiang94**
Changed Parser methods to class methods (i.e. static) 
Add an instance-level member int sequenceNumber and a class-level variable int nextSequenceNumber to the Person class.

### CODE REVIEW

**Comment by YongSiang94** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -84,4 +88,12 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+	public int getSequenceNumber() {
+		return sequenceNumber;
+	}
+
+	public void setSequenceNumber(int sequenceNumber) {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Person.java:95]: No need to used set method.

File: src/seedu/addressbook/data/person/Person.java [hunk 2]
```diff
@@ -84,4 +88,12 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+	public int getSequenceNumber() {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Person.java:91]: You can put method description like `Returns ...`

File: src/seedu/addressbook/data/person/Person.java [hunk 3]
```diff
@@ -9,12 +9,14 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
-
+	
+	private static int nextSequenceNumber = 1;
+	private int sequenceNumber;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Person.java:14]: Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)

File: src/seedu/addressbook/data/person/Person.java [hunk 4]
```diff
@@ -25,6 +27,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.setSequenceNumber(nextSequenceNumber);
+        nextSequenceNumber ++;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Person.java:31]: You can just write it as `SequenceNumber = nextSequenceNumber++;`

**Comment by jeffryhartanto** [comment]
@YongSiang94 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**